authors/settings/staging.py

```diff
@@ -0,0 +1,15 @@
+from authors.settings.base import *
```

Comment:
How different are the staging settings from the development settings?

Reply:
The heroku postgres connection strings are different.